.github/workflows/build.yml

```diff
@@ -5,8 +5,13 @@ on:
 jobs:
   build:
     runs-on: [self-hosted, linux, x64]
+    strategy:
+      matrix:
+        variant: [aws-k8s-1.15, aws-k8s-1.16]
```

Comment:
Any reason, other than total run time, to not include `aws-dev` in this list?

Reply:
I'm somewhat reluctant to promote it as a "Tier 1" target which is how I view this list. All of the packages it needs still get built, so we'd catch docker build failures for example.

Reply:
Gotcha. I think we should still be building the variant as much as possible - perhaps we could include it in a non-blocking/non-critical way where it isn't required for PR to pass?